.gitignore

```diff
@@ -1,3 +1,6 @@
+# User generated config files
```

Comment:
Maybe change this to "Config files generated from testing script ``pull_cfg..." 

This script has a path from your home in it ...